### PR TITLE
export appealTypes

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
I noticed this same array was being made in multiple places so now it's just exported from the appeals helpers file.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/dashboard/containers/ClaimsAppealsWidget.jsx [hunk 1]
```diff
@@ -246,8 +243,6 @@ const mapStateToProps = state => {
     appealsLoading: claimsV2Root.appealsLoading,
     claimsAppealsCount,
     claimsAppealsList,
-    consolidatedModal: claimsRoot.consolidatedModal,
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/containers/ClaimsAppealsWidget.jsx:249]: these two props were _never_ used, as far as I can tell from my git spelunking.

File: src/applications/personalization/dashboard/containers/ClaimsAppealsWidget.jsx [hunk 2]
```diff
@@ -246,8 +243,6 @@ const mapStateToProps = state => {
...
-    show30DayNotice: claimsRoot.show30DayNotice,
     synced: claimsState.claimSync.synced,
```
Comment (Mottie) [inline src/applications/personalization/dashboard/containers/ClaimsAppealsWidget.jsx:246]: Are the 3 deleted lines above intentional and within scope?

**Review by Mottie — APPROVED** [review]

File: src/applications/personalization/dashboard/containers/ClaimsAppealsWidget.jsx (hunk 2 above)
Comment (erikphansen) [reply src/applications/personalization/dashboard/containers/ClaimsAppealsWidget.jsx:246]: within scope? given that there isn't a ticket for this work, yes :-D

**Review by Mottie — APPROVED** [review]

**Review by sanlouise — APPROVED** [review]

**Review by mdewey — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**